index.html

```diff
@@ -152,7 +152,7 @@
 <section id="sotd">
 	<p>The Accessible Rich Internet Applications Working Group seeks feedback on any aspect of the specification. When submitting feedback, please consider issues in the context of the companion documents. To comment, <a href="https://github.com/w3c/aria/issues/new">file an issue in the <abbr title="World Wide Web Consortium">W3C</abbr> ARIA GitHub repository</a>. If this is not feasible, send email to <a href="mailto:public-aria@w3.org?subject=Comment%20on%20WAI-ARIA%201.2">public-aria@w3.org</a> (<a href="http://lists.w3.org/Archives/Public/public-aria/">comment archive</a>). In-progress updates to the document may be viewed in the <a href="http://w3c.github.io/aria/">publicly visible editors' draft</a>.</p>
 </section>
-<section class="informative" id="introduction">
+<section class="informative" id="introduction" data-cite="dom">
```

Comment:
Can we start making these concepts normative? As they are used by the rest of the spec in normative parts, that would make sense. 

 
```suggestion
<section  id="introduction" data-cite="dom">
```

Reply:
Alternatively, we could split this into normative, informative terms if need be... but probably just fine to treat terms as normative.

Reply:
This is a bigger issue. If we are to consider we need a new issue to be filed and we will deal with it there. This PR should be as limited as we can to unblock or we will never merge it

Reply:
Filed https://github.com/w3c/aria/issues/1513